src/passkey_child/passkey_child_common.c

```diff
@@ -890,3 +910,47 @@ verify_assert_data(struct passkey_data *data)
 
     return ret;
 }
+
+errno_t
+preflight(struct passkey_data *data, int timeout)
```

Comment:
`select_authenticator()` /  `preflight()` could take pointer to `const struct passkey_data`